### ISSUE TITLE
remove prototype a11y code from this sim

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/forces-and-motion-basics#275]
Noticed in https://github.com/phetsims/tasks/issues/984, there is a11y code that was used in initial prototypes that should be removed. Our design practices for a11y have changed since this code was added so I see little benefit in keeping it. And if there is a reason to keep any of the initial designs, the implementation should be redone anyway because the project a11y infrastructure has changed since this was originally added.

For reference just in case, here is a link to one of the initial prototype designs:
https://phet-dev.colorado.edu/html/forces-and-motion-basics/2.1.0-accessible-instance.1/forces-and-motion-basics_en.html?accessibility

**Comment by jessegreenberg** [comment]
OK, it is been removed and cleaned up. Sim is working, unit tests are passing, sim is fuzzing. I think this can be closed.
